spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -134,6 +134,13 @@ def assert_specific_vm_powered_on
     )
 
     expect(v.ext_management_system).to eq(@ems)
+    expect(v.orchestration_stack).to have_attributes(
```

Comment:
You should probably have a separate `assert_specific_orchestration_stack` where you set `@orchestration_stack` and check for attributes so here you can just say `expect().to eq(@orchestration_stack)`